cypress/support/index.js

```diff
@@ -42,6 +42,7 @@ export const generatePatient = () => {
   patient.dobForPatientLink = patient.dob.format("MM/DD/YYYY");
   patient.phone = "(800) 232-4636";
   patient.address = "736 Jackson PI NW";
+  patient.city = "Cleveland"
```

Comment:
This needs to be set to `Washington`, as a nod to the original authors of the SR

Reply:
So having it as Washington causes the test to fail because we're testing for the smarty streets modal to pop and it doesn't do that when the address is completely valid.